site/src/atomics/spacing.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+- `<css-property-name>-<value>-<screensize>`. For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md)
+
+Accepted property names
```

Comment:
Full sentences? Rearrange to include the list checked into the bugfix PR?

Reply:
`The following <property-names|values|screensizes> are supported.`?

Reply:
Updated

Reply:
For our lists of accepted things here, let's just make sure use the same format.

1. If we're going to use a list, use a list (not a list of one item like current accepted values / accepted screensizes.
2. If we're not, just list supported values in a paragraph.
3. List all values, (not regexpy syntax)

Either

```
- `margin`
- `padding`
- `margin-bottom`
- etc

Accepted sizes

- `none`
- `xs`
- etc
```

Or we just list them in inline text. Whatever we pick, let's try and use the same format.